docs/docs/API-Reference/api-reference-api-examples.md

```diff
@@ -642,87 +651,268 @@ curl -X GET \
 ```json
 {
   "files": [
-    "2024-12-30_15-19-43_the_oscar_award.csv"
+    "2024-12-30_15-19-43_your_file.txt"
   ]
 }
 ```
 
   </TabItem>
 </Tabs>
 
-### Download file
+### Download file (v1)
 
-Download a specific file for a given flow.
+Download a specific file from a flow.
 
-To look up the file name in Langflow, use the `/list` endpoint.
+<Tabs>
+  <TabItem value="curl" label="curl" default>
 
-This example downloads the `2024-12-30_15-19-43_the_oscar_award.csv` file from Langflow to a file named `output-file.csv`.
+```bash
+curl -X GET \
+  "$LANGFLOW_URL/api/v1/files/download/$FLOW_ID/2024-12-30_15-19-43_your_file.txt" \
+  -H "accept: application/json" \
+  --output downloaded_file.txt
+```
 
-The `--output` flag is optional.
+  </TabItem>
+  <TabItem value="result" label="Result">
+
+```text
+File contents downloaded to downloaded_file.txt
+```
+
+  </TabItem>
+</Tabs>
+
+### Delete file (v1)
+
+Delete a specific file from a flow.
 
 <Tabs>
   <TabItem value="curl" label="curl" default>
 
 ```bash
-curl -X GET \
-  "$LANGFLOW_URL/api/v1/files/download/$FLOW_ID/2024-12-30_15-19-43_the_oscar_award.csv" \
+curl -X DELETE \
+  "$LANGFLOW_URL/api/v1/files/delete/$FLOW_ID/2024-12-30_15-19-43_your_file.txt" \
+  -H "accept: application/json"
+```
+
+  </TabItem>
+  <TabItem value="result" label="Result">
+
+```json
+{
+  "message": "File 2024-12-30_15-19-43_your_file.txt deleted successfully"
+}
+```
+
+  </TabItem>
+</Tabs>
+
+## Files/V2 Endpoints
```

Comment:
Use a lowercse e in endpoints. Also, you should not have headings without any body text. Add body text to describe what this section is about.